src/external/libnfsidmap.m4

```diff
@@ -17,4 +17,13 @@ AS_IF([test x"$with_nfsv4_idmap" = xyes], [
             [AC_MSG_ERROR([libnfsidmap header files are not installed]
 If you want to build sssd without nfs idmap pluging then specify
 --without-nfsv4-idmapd-plugin when running configure.)])])
+
+    AC_CHECK_HEADERS([nfsidmap_plugin.h], [], [],
+        [#ifdef HAVE_STDLIB_H
+# include <stdlib.h>
+#endif
+#ifdef HAVE_STDINT_H
+# include <stdint.h>
+#endif
+#include <nfsidmap.h>])
```

Comment:
Shouldn't this say nfsidmap_plugin.h?

Reply:
I do not think so and documentation says the same
https://www.gnu.org/savannah-checkouts/gnu/autoconf/manual/autoconf-2.69/html_node/Generic-Headers.html#Generic-Headers